apriltag_ros/include/apriltag_ros/continuous_detector.h

```diff
@@ -81,6 +103,22 @@ class ContinuousDetector: public nodelet::Nodelet
 
   ros::ServiceServer refresh_params_service_;
   bool refreshParamsCallback(std_srvs::Empty::Request& req, std_srvs::Empty::Response& res);
+
+  // publishes an empty detection
+  void publishEmptyDetection(const std_msgs::Header& header);
+
+  // checks if a tag is in the FOV of the camera given the tag pose in optical frame
+  bool isTagInFOV(const tf2::Transform& tag_pose) const;
+
+  // checks if a tag is within the minimum and maximum detection range
+  bool isTagInDetectionRange(const tf2::Transform& tag_pose) const;
+
+  // get the target pose in the base frame
+  void targetPoseCallback(const geometry_msgs::PoseStamped& pose_msg);
+
+  // TODO (npalmar): fix this when it gets moved to cpr_tf2_ros
```

Comment:
Planning on getting rid of this and maybe the tf_listener depend by moving that into camera_transformer